website/docs/en/config/performance/build-cache.mdx

```diff
@@ -87,3 +88,22 @@ export default defineConfig({
   },
 });
 ```
+
+### buildDependencies
+
+- **Type:** `string[]`
+- **Default:** `undefined`
```

Comment:
Should we tell users that this array will merge with the default array?

Reply:
done